docs/develop-a-squid/squid-processor/store-interface.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 4
+description: >-
+  Details on the store interface exposed by the handler context
+---
+
+# Store interface
+
+`Store` is a generic interface exposed by `BatchContext.store` to the handlers. The concrete type is inferred from the `Database` argument of the `BatchProcessor.run` method:
```

Comment:
Do you want to make this specific to the Batch Processor and Batch Context? We could say "*Context", because it works the same way for the others...